test/unit/membership_test.js

```diff
@@ -204,7 +191,7 @@ testRingpop('cannot evict self', function t(deps, assert) {
 testRingpop('generate checksums string preserves order of members', function t(deps, assert) {
     var membership = deps.membership;
 
-    for (var i = 0; i < 100; i++) {
+    for (var i = 1; i < 100; i++) {
```

Comment:
correct. I'll add a comment
(side note: this change is in fact not necessary because calling makeAlive for the local member will just re-incarnate the local member again)